site/docs/v1/tech/apis/_login-response-codes.adoc

```diff
@@ -35,7 +35,7 @@ include::../../../src/json/login/login-change-password-response.json[]
 Prior to version `1.27.0`, this status code was not returned, and you will see a `200` instead.
 
 |242
-|The user was authenticated successfully. The user has two factor authentication enabled, the response will contain the `twoFactorId` to be used
+|The user was authenticated successfully. The user has two factor authentication enabled. Since version `1.42.0`, this status code is returned when two factor authentication is required. The response will contain the `twoFactorId` to be used
```

Comment:
```suggestion
|The user was authenticated successfully. The user has two factor authentication enabled. Since version `1.42.0`, this status code is also returned when two factor authentication is required. The response will contain the `twoFactorId` to be used
```

I didn't look at the code, but am guessing we return a `242` in either case.